usefulUtilities/spinnableWheel/spinnableWheelServer.js

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable no-magic-numbers */
+//
+// spinabbleWheelServer.js
+// 
+// Created by Zach Fox on 2019-04-19
+// Copyright High Fidelity 2019
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+/* global console */
+
+(function() {
+    var that;
+    var CHECK_SPINNING_TIMEOUT_MS = 100;
+    var RAD_PER_DEG = Math.PI / 180;
+    var SMALL_Z_ANGULAR_VELOCITY_RAD_PER_SEC = 15 * RAD_PER_DEG;
+    var MIN_ANGULAR_VELOCITY_RAD_PER_SEC = 40;
+    var MAX_ANGULAR_VELOCITY_RAD_PER_SEC = 100;
+
```

Comment:
Can you add some function comments?

Reply:
yeahhhhh i can, thank you for calling it out :)